source/Calamari.Tests/Fixtures/Integration/Scripting/PowerShellScriptEngineFixture.cs

```diff
@@ -12,6 +12,7 @@ public class PowerShellScriptEngineFixture : ScriptEngineFixtureBase
     {
         [Test]
         [RequiresNonFreeBSDPlatform]
+        [RequiresNonARM64Bit]
```

Comment:
Would be good to have a description here
```suggestion
        [RequiresNonARM64Bit("As of June 2020, PowerShell Core for ARM is still an experimental release that is unsupported.")]
```
etc (so we know why we're excluding it)